src/Tribe/Views/V2/Hooks.php

```diff
@@ -581,4 +582,50 @@ public function filter_events_properties( $event ) {
 
 		return $this->container->make( Template\Event::class )->filter_event_properties( $event );
 	}
+
+	/**
+	 * Indicates if month view cache should be enabled or not.
+	 *
+	 * If the month view cache setting itself is not enabled (or not set) then this
+	 * method will always return false.
+	 *
+	 * In other cases, the default rules are to cache everything in the 2 months past
+	 * to 12 months in the future range. This policy can be refined or replaced via
+	 * the 'tribe_events_enable_month_view_cache' filter hook.
+	 *
+	 * @TODO: adjust this so it makes sense and move to an appropriate class
+	 *
+	 * @return bool
+	 */
+	protected function should_enable_month_view_cache( $context ) {
+		$event_date = $context->get( 'event_date' );
+
+		// Respect the month view caching setting
+		if ( ! tribe_get_option( 'enable_month_view_cache', true ) ) {
+			return false;
+		}
+
+		// Default to always caching the current month
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 44f2502f2df2f3d576b638c6dff76f13</sub>